gpiozero/output_devices.py

```diff
@@ -503,6 +503,243 @@ def _blink_device(
+            pin_factory=None):
+        super(PWMBuzzer, self).__init__(
+            pin=pin, active_high=active_high, initial_value=initial_value, frequency=frequency,
+            pin_factory=pin_factory
+        )
+
+        self.notes = {
+            """
+            Dictionary containing musical notes which can be played through the PWMBuzzer.
+            """
+            'A0': 27.50,
+            'AS0': 29.14,
+            'B0': 30.87,
+            'C1': 32.70,
+            'CS1': 34.65,
+            'D1': 36.71,
+            'DS1': 38.89,
+            'E1': 41.20,
+            'F1': 43.65,
+            'FS1': 46.25,
+            'G1': 49.0,
+            'GS1': 51.91,
+            'A1': 55.00,
+            'AS1': 58.27,
+            'B1': 61.74,
+            'C2': 65.41,
+            'CS2': 69.30,
+            'D2': 73.42,
+            'DS2': 77.78,
+            'E2': 82.41,
+            'F2': 87.31,
+            'FS2': 92.50,
+            'G2': 98.0,
+            'GS2': 103.83,
+            'A2': 110.0,
+            'AS2': 116.54,
+            'B2': 123.47,
+            'C3': 130.81,
+            'CS3': 138.59,
+            'D3': 146.83,
+            'DS3': 155.56,
+            'E3': 164.81,
+            'F3': 174.61,
+            'FS3': 185.0,
+            'G3': 196.0,
+            'GS3': 207.65,
+            'A3': 220.0,
+            'AS3': 233.08,
+            'B3': 246.94,
+            'C4': 261.63,
+            'CS4': 277.18,
+            'D4': 293.66,
+            'DS4': 311.13,
+            'E4': 329.63,
+            'F4': 349.23,
+            'FS4': 369.99,
+            'G4': 392.0,
+            'GS4': 415.30,
+            'A4': 440.0,
+            'AS4': 466.16,
+            'B4': 493.88,
+            'C5': 523.25,
+            'CS5': 554.37,
+            'D5': 587.33,
+            'DS5': 622.25,
+            'E5': 659.26,
+            'F5': 698.46,
+            'FS5': 739.99,
+            'G5': 783.99,
+            'GS5': 830.61,
+            'A5': 880.00,
+            'AS5': 932.33,
+            'B5': 987.77,
+            'C6': 1046.50,
+            'CS6': 1108.73,
+            'D6': 1174.66,
+            'DS6': 1244.51,
+            'E6': 1318.51,
+            'F6': 1396.91,
+            'FS6': 1479.98,
+            'G6': 1567.98,
+            'GS6': 1661.22,
+            'A6': 1760.0,
+            'AS6': 1864.66,
+            'B6': 1975.53,
+            'C7': 2093.0,
+            'CS7': 2217.46,
+            'D7': 2349.32,
+            'DS7': 2489.02,
+            'E7': 2637.02,
+            'F7': 2793.82,
+            'FS7': 2959.96,
+            'G7': 3135.96,
+            'GS7': 3322.44,
+            'A7': 3520.0,
+            'AS7': 3729.31,
+            'B7': 3951.07,
+            'C8': 4186.01,
+        }
+
+        self.midi_notes = {
+            """
+            Dictionary containing MIDI notes which can be played through the PWMBuzzer.
+            """
+            '21': 27.50,
```

Comment:
In order for the `play` function to work for midi notes, I think you'll need to remove the quotes from the keys in this dictionary, i.e. `21: 27.50,` etc.

Reply:
Removed the quotes around the MIDI notes.